server/methods/alexaApiCall.js

```diff
@@ -0,0 +1,38 @@
+import { Meteor } from 'meteor/meteor';
+import { HTTP } from 'meteor/http';
+import { check } from 'meteor/check';
+
+import { settings } from '../../app/settings';
+
+Meteor.methods({
+	registerAlexaUser(serverurl, servername) {
+		check(serverurl, String);
+		check(servername, String);
+
+		const userid = Meteor.userId();
+
+		if (!userid) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', {
+				method: 'registerAlexaUser',
+			});
+		}
+
+		const tokenName = `alexa-${ servername }`;
+		const token = Meteor.runAsUser(this.userId, () => Meteor.call('personalAccessTokens:generateToken', { tokenName }));
```

Comment:
I don't think you need to use `runAsUser` here since the method is already running with the credentials of the logged user. Can you test?

Reply:
ok I will check

Reply:
yaah it does work without `runAsUser`.